Source/Renderer/ContextLimits.js

```diff
@@ -0,0 +1,271 @@
+/*global define*/
+define([
+        '../Core/defineProperties'
+    ], function(
+        defineProperties) {
+    "use strict";
+
+    /**
+     * @private
+     */
+    var ContextLimits = function(){};
```

Comment:
Despite whatever I said before, since everything is static, this is probably preferred:

```javascript
var ContextLimits = {
   _maximumCombinedTextureImageUnits : 0
   // ...
};
```